deploy/chart/templates/0000_30_19-operatorsource-default.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: "marketplace.redhat.com/v1alpha1"
+kind: "OperatorSource"
+metadata:
+  name: "marketplace-operators"
+  namespace: "openshift-operators"
```

Comment:
this should be `namespace: {{ .Values.operator_namespace }}`

Reply:
@ecordell That sets the namespace in the upstream/manifests/ folder to "operators" instead of "openshift-operators". Is that intentional?

Reply:
To follow up on this, the upstream/manifests/ folder is for non-openshift upstream kubernetes distributions which is the reason for the namespace change.

I have addressed the requested change and updated the pr to reflect it.